test/cloudformation/iam_cloudformation.yaml

```diff
@@ -90,6 +91,8 @@ Resources:
             Action:
               - iam:CreateOpenIDConnectProvider
               - iam:DeleteOpenIDConnectProvider
+              - iam:ListOpenIDConnectProviders
```

Comment:
Is there a permission that is associated with listing the OIDC provider tags since we are also doing that?